### ISSUE TITLE
fail to install in win7 x64 with sp1 beta

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rakibtg/gdipp#88]
```
Which program triggers the problem? Specify the filename if possible.
windows 7

What steps will reproduce the problem?
1. failed to install, both in service and registration method
2.
3.

What is the expected output? What do you see instead?
with service install, the system said service failed to started
with registration install, the system keeps saying can not location gdimm

What version of gdipp (including renderer)? On which operating system?
0.82

Please provide any additional information below. Screenshots always help.
actually, I install 0.81 before installing sp1 beta, and it works fine. 
however, after the failure of 0.82, I tried to install 0.81 and it failed too.
Another thing that I notice during these days is that with gdipp installed, IE8 
seems to crash very frequently.

```

Original issue reported on code.google.com by `yep...@gmail.com` on 9 Jul 2010 at 3:06

**Comment by GoogleCodeExporter** [comment]
```
Maybe you want to try to place gdipp_common_64.dll in C:\Windows\SysWOW64 and 
gdipp_common_32.dll in System32. Maybe that will help!
```

Original comment by `flyrobin...@gmail.com` on 15 Jul 2010 at 10:47

**Comment by GoogleCodeExporter** [comment]
```
I just had this same issue with Win 7 64-bit - no service pack. With service 
install, said service failed to start. 0.8.2 
```

Original comment by `harris.t...@gmail.com` on 20 Jul 2010 at 9:48

**Comment by GoogleCodeExporter** [comment]
```
well, the copy file method did not work for me. still can't use.
looks like the new version is working on directwrite render, very looking 
forward
```

Original comment by `yep...@gmail.com` on 21 Jul 2010 at 2:48

**Comment by GoogleCodeExporter** [comment]
```
I'm not sure why so many people report that the basic installation and 
rendering are malfunction, and I cannot reproduce at all. In short, I may not 
help unless someone could report more "technical" information.

gdipp is not tested under SP1. Maybe the test will be carried out until the SP1 
official release.
```

Original comment by `crendk...@gmail.com` on 8 Aug 2010 at 12:08

**Comment by GoogleCodeExporter** [comment]
```
Thank you for the comments. It's was kinda wired cause I retried couples days 
ago and it worked out of box! My guess is maybe the MS's Tuesday's path did 
something, or may just random thing. 
Anyhow, will 0.9 version include directwrite support?
```

Original comment by `yep...@gmail.com` on 17 Aug 2010 at 7:09

**Comment by GoogleCodeExporter** [comment]
```
DirectWrite will be included in 0.9 with even better performance, thanks to the 
new glyph run caching mechanism.
```

Original comment by `crendk...@gmail.com` on 18 Aug 2010 at 12:52
- Changed state: **Done**

**Comment by GoogleCodeExporter** [comment]
```
awesome! crendking you are amazing!
```

Original comment by `yep...@gmail.com` on 18 Aug 2010 at 2:19
